src/App.js

```diff
@@ -1,18 +1,28 @@
 import React from 'react';
 
```

Comment:
`var` generally doesn't get used unless anymore unless for very specific reasons. There are two other declarations that are more commonly used. Do a little research and see which one better fits your use case.

Reply:
changed to let